### PR TITLE
Add new characteristics to statistics integration

### PR DESCRIPTION
**Pull request by ThomDietrich**
## Proposed change

Document new statistics characteristics from https://github.com/home-assistant/core/pull/62631

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/62631
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/statistics.markdown [hunk 1]
```diff
@@ -44,10 +44,12 @@ The following characteristics are supported for `sensor` source sensors:
 | `average_timeless` | The average value of stored measurements. This method assumes that all measurements are equally spaced and, therefore, time is ignored and a simple average of values is computed. Equal to `mean`.
 | `change_sample` | The average change per sample. The difference between the oldest and newest measurement is divided by the number of in-between measurements (n-1).
 | `change_second` | The average change per second. The difference between the oldest and newest measurement is divided by seconds between them.
-| `change` | The difference between the oldest and newest measurement stored.
+| `change` | The difference between the oldest and newest measurement.
 | `count` | The number of stored source sensor readings. This number is limited by `sampling_size` and can be low within the bounds of `max_age`.
-| `datetime_newest` | The timestamp of the newest measurement stored.
-| `datetime_oldest` | The timestamp of the oldest measurement stored.
+| `datetime_newest` | The timestamp of the newest measurement.
```
Comment (emontnemery) [inline source/_integrations/statistics.markdown:49]: In this case, should the source sensor have device class timestamp? Maybe add a column to the table with sensor type, or (and that's probably clearer) add a new section "Timestamp source sensor"?

Comment (ThomDietrich) [reply source/_integrations/statistics.markdown:49]: Not sure I understand. The two tables are separated based on the source sensor, with numeric or binary values in nature. In the case here the source sensor is any `sensor.x` and the four `datetime_*` characteristics produce a timestamp as output. I agree with your device class question, and yes that is handled by the statistics component, but how is that relevant here I don't know :)

Comment (emontnemery) [reply source/_integrations/statistics.markdown:49]: I misunderstood what this does, never mind.

File: source/_integrations/statistics.markdown [hunk 2]
```diff
@@ -44,10 +44,12 @@ The following characteristics are supported for `sensor` source sensors:
...
+| `datetime_oldest` | The timestamp of the oldest measurement.
+| `datetime_value_max` | The timestamp of the biggest value among the number of measurements.
```
Comment (emontnemery) [inline source/_integrations/statistics.markdown]: *among the number of measurements* is a bit strange, can we try to rephrase it?

**Review by emontnemery — APPROVED** [review]
Thanks, @ThomDietrich 👍

**Pull request merged**